ts/session/apis/snode_api/signature/groupSignature.ts

```diff
@@ -0,0 +1,326 @@
+import {
+  GroupPubkeyType,
+  PubkeyType,
+  Uint8ArrayLen100,
+  Uint8ArrayLen64,
+  UserGroupsGet,
+  WithGroupPubkey,
+} from 'libsession_util_nodejs';
+import { isEmpty, isString } from 'lodash';
+import { MetaGroupWrapperActions } from '../../../../webworker/workers/browser/libsession_worker_interface';
+import { GroupUpdateInviteMessage } from '../../../messages/outgoing/controlMessage/group_v2/to_user/GroupUpdateInviteMessage';
+import { GroupUpdatePromoteMessage } from '../../../messages/outgoing/controlMessage/group_v2/to_user/GroupUpdatePromoteMessage';
+import { StringUtils, UserUtils } from '../../../utils';
+import { fromUInt8ArrayToBase64, stringToUint8Array } from '../../../utils/String';
+import { PreConditionFailed } from '../../../utils/errors';
+import { SnodeNamespacesGroup } from '../namespaces';
+import { SignedGroupHashesParams, WithMessagesHashes, WithShortenOrExtend } from '../types';
+import { SignatureShared } from './signatureShared';
+import { SnodeSignatureResult } from './snodeSignatures';
+import { getSodiumRenderer } from '../../../crypto';
+import { NetworkTime } from '../../../../util/NetworkTime';
+
+async function getGroupInviteMessage({
+  groupName,
+  member,
+  secretKey,
+  groupPk,
+}: {
+  member: PubkeyType;
+  groupName: string;
+  secretKey: Uint8ArrayLen64; // len 64
```

Comment:
comment feels redundant

Reply:
moved to docString, the `Uint8ArrayLen64` does show as `Uint8Array` on hover.